CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ Changes since v2.13
 - Deciders and reviewers can now be invited via email. (#2040)
   - New `invite-decider` and `invite-reviewer` commands in the API & UI
   - Commands are available to the handler on submitted applications. See [permission table](docs/application-permissions.md).
+- The first version of REMS user manual
```

Comment:
could link to the manual like this: `[manual](manual/)`

Reply:
Done